deepmd/pt/train/training.py

```diff
@@ -486,60 +487,114 @@ def get_loss(loss_params, start_lr, _ntypes, _model):
+                            model_key,
+                            model_key_from,
+                            new_state_dict,
+                            state_dict,
+                            target_state_dict,
+                            _new_fitting=new_fitting,
+                        )
+                    else:
+                        for model_key in self.model_keys:
+                            if model_key in self.finetune_links:
+                                model_key_from = self.finetune_links[model_key]
+                                new_fitting = model_params["model_dict"][model_key].pop(
+                                    "new_fitting", False
+                                )
+                            else:
+                                model_key_from = model_key
+                                new_fitting = False
+                            update_single_finetune_params(
+                                model_key,
+                                model_key_from,
+                                new_state_dict,
+                                state_dict,
+                                target_state_dict,
+                                _new_fitting=new_fitting,
                             )
                     state_dict = new_state_dict
-                if finetune_model is not None:
                     state_dict["_extra_state"] = self.wrapper.state_dict()[
                         "_extra_state"
                     ]
-
                 self.wrapper.load_state_dict(state_dict)
-                # finetune
-                if finetune_model is not None and model_params["fitting_net"].get(
-                    "type", "ener"
-                ) in ["ener", "direct_force_ener", "atten_vec_lcc"]:
+
+                def single_model_finetune(
+                    _model,
+                    _model_params,
+                    _sample_func,
+                ):
                     old_type_map, new_type_map = (
-                        model_params["type_map"],
-                        model_params["new_type_map"],
+                        _model_params["type_map"],
+                        _model_params["new_type_map"],
                     )
-                    # TODO: need an interface instead of fetching fitting_net!!!!!!!!!
-                    if hasattr(self.model, "atomic_model") and hasattr(
-                        self.model.atomic_model, "fitting_net"
-                    ):
-                        self.model.atomic_model.fitting_net.change_energy_bias(
-                            config,
-                            self.model,
-                            old_type_map,
-                            new_type_map,
-                            ntest=ntest,
-                            bias_shift=model_params.get("bias_shift", "delta"),
+                    if isinstance(_model, EnergyModel):
+                        _model.atomic_model.change_out_bias(
```

Comment:
bad design. you are accessing the datamember of the model!.  we should provide the `change_out_bias` method for model that is forwarding the atomic model's `change_out_bias`